frontend/src/features/public-form/components/SectionSidebar/SectionSidebar.tsx

```diff
@@ -42,6 +44,14 @@ export const SectionSidebar = (): JSX.Element => {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [miniHeaderRef?.current?.clientHeight])
 
+  const navigatedSection = useMemo(() => {
+    if (!navigatedSectionId || !form) return
+    if (navigatedSectionId === PUBLICFORM_INSTRUCTIONS_SECTIONID) {
+      return { title: 'Instructions', description: form.startPage.paragraph }
```

Comment:
Just checking: does this need an additional compatibility property `_id: PUBLICFORM_INSTRUCTIONS_SECTIONID`?

Reply:
its not exactly needed because `navigatedSection` doesn't have the type of form fields. we only ever use the title and description fields from `navigatedSection`. Maybe it's better to say this instead, for clarity?

```typescript
const {title: navigatedSectionTitle, description: navigatedSectionDescription } = useMemo(() => {...

Reply:
it didn't seem that it was needed, i guess ts is smart enough to know that only the title and description fields were referenced, haha